### DISCUSSION TITLE
italian update

### DISCUSSION THREAD

**Discussion by dantecarraro** [aclap-dev/video-downloadhelper#864] [Language & Translation]
[it.txt](https://github.com/user-attachments/files/15809878/it.txt)

**Comment by giadone81** [comment]
[
[it.txt](https://github.com/user-attachments/files/15838649/it.txt)
](url)

**Comment by paulrouget** [comment]
Sorry we already got a translation for that language 🙏

I recommend that you try the upcoming version (v9.1 which should be released next week) see if the translation is correct.

If you want to keep track of the current translation status of your language, I recommend following this github repository: https://github.com/aclap-dev/vdhlocales
